allCombined.py

```diff
@@ -91,6 +91,8 @@ def getResult(universityName,url,userName,password):
 			result= s[s.find("Status")+1:s.find(" <img")]
 			result=result[7:]
 		elif(getType==1):
+			result= s[s.find("Status")+1:s.find(" <img")]
+			result=result[7:]
```

Comment:
Before it was not working because these couple of lines were missing. When I've added them back it was working fine. Also I saw on edulix where many users were telling that the green tick is missing for Materials Received. So I thought it would be better to track that as well. Feel free to remove those, if you don't find it that useful. As you said, it only tells whether decision is out or not, nothing more than that.